pyapp/cookiecutter.json

```diff
@@ -16,7 +16,7 @@
     "docker": ["no", "yes"],
     "__postgres_port": "{{ random_port_number() }}",
     "__docker_namespace": "{{ cookiecutter.github_owner }}",
-    "__docker_network": "{{ cookiecutter.package_name }}_default",
+    "__docker_network": "{{ cookiecutter.slug }}_default",
```

Comment:
Whoops, randomly spotted a mistake here that was causing `make docker-run` to fail in the test projects because the docker network name was wrong